### DISCUSSION TITLE
Plotting Pole Figure for Family of Planes {hkl} and Directions <uvw>

### DISCUSSION THREAD

**Discussion by KatieBillings** [mtex-toolbox/mtex#2230] [Ask Anything] [answered]
Hello all,
I am trying to plot pole figures for some calcite data. I would like to plot pole figures for (0001) {10-14} <-12-10> and <20-2-1>, however I am unsure how to plot the symmetrical planes, denoted by {hkl}, and symmetrical directions, denoted by < uvw >, in mtex. I suspect I'll need to make use of the 'symmetrise' function, though I've had no luck getting it to plot correctly.  Please let me know if anyone has any suggestions. Thank you!

Here is how I have attempted to plot my pole figures:
```matlab
C_CS = crystalSymmetry('-3m1', [5 5 17], 'X||a*', 'Y||b', 'Z||c*', 'mineral', 'Calcite', 'color', [0.6 0.8 0]);
h = {Miller(0,0,0,1,'hkil',C_CS), Miller(1,0,-1,4,'hkil',C_CS), (Miller(-1,2,-1,0,'uvtw',C_CS)),Miller(2,0,-2,-1,'uvtw',C_CS)};
C_Or = meanOr_data;
ori = calcDensity(C_Or,'halfwidth',10*degree);
figure
plotPDF(ori,h,'hemisphere','lower','projection','earea','silent','planes','contourf')
```
Which generates the following plot:
![Screenshot 2024-09-24 123038](https://github.com/user-attachments/assets/8acddcd6-a2e7-4a7d-9bda-8d08ea76fff1)

**Comment by lfgmorales** [comment, marked as answer]
Hi katie,

What do you mean by plotting the “symmetrical planes”? The crystal symmetry is intrinsic on your definition of CS, calcite is trigonal, so when you say {10-14} it will plot the group of pole to planes symmetrically equivalent in that pole figure, to me your pole figures look good, you see one maximum concentration of 0001 and three of 10-14, showinf the trigonal symmetry. But maybe I misunderstood the question

Cheers

Luiz


Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 24 Sep 2024, at 20:33, Katie Billings ***@***.***> wrote:

﻿

Hello all,
I am trying to plot pole figures for some calcite data. I would like to plot pole figures for (0001) {10-14} <-12-10> and <20-2-1>, however I am unsure how to plot the symmetrical planes, denoted by {hkl}, and symmetrical directions, denoted by < uvw >, in mtex. I suspect I'll need to make use of the 'symmetrise' function, though I've had no luck getting it to plot correctly. Please let me know if anyone has any suggestions. Thank you!

Here is how I have attempted to plot my pole figures:
C_CS = crystalSymmetry('-3m1', [5 5 17], 'X||a*', 'Y||b', 'Z||c*', 'mineral', 'Calcite', 'color', [0.6 0.8 0]);
h = {Miller(0,0,0,1,'hkil',C_CS), Miller(1,0,-1,4,'hkil',C_CS), (Miller(-1,2,-1,0,'uvtw',C_CS)),Miller(2,0,-2,-1,'uvtw',C_CS)};
C_Or = meanOr_data;
ori = calcDensity(C_Or,'halfwidth',10*degree);
figure
plotPDF(ori,h,'hemisphere','lower','projection','earea','silent','planes','contourf')

Which generates the following plot:
Screenshot.2024-09-24.123038.png (view on web)<https://github.com/user-attachments/assets/8acddcd6-a2e7-4a7d-9bda-8d08ea76fff1>

**Reply by KatieBillings** [reply]
Hi Luiz,
Thank you for your helpful response. I understand now that its already accounting for the symmetry when its plotting. I was confused since the notation doesn't change in the output. Thank you for your help!

For anyone else confused, it is clearer that it is plotting correctly when I plot a single randomly oriented grain:
![Screenshot 2024-09-26 143414](https://github.com/user-attachments/assets/2eb54f31-73f8-49c3-a6b7-6537d4fd15a3)
